spot-client/src/spot-remote/app-state/actions.js

```diff
@@ -139,6 +140,7 @@ export function connectToSpotTV(joinCode, shareMode) {
                 isSpotTv: false,
                 isPairingPermanent: backend && backend.isPairingPermanent()
             }));
+            dispatch(reconnectScheduleUpdate(false));
```

Comment:
I am wondering if it would be better for safety to also do this call in `disconnectFromSpotTV`. It seems to be duplicating other cleanup logics as well.